govcd/catalog_test.go

```diff
@@ -207,6 +207,55 @@ func doesCatalogExist(check *C, org *AdminOrg) {
 	check.Assert(err, NotNil)
 }
 
+// Creates a Catalog, uploads a vApp template to it, renames it, retrieves it
+// using the updated name and checks if it has the same vApp template.
+// If it doesn't the assertion fails.
+func (vcd *TestVCD) Test_RenameCatalog(check *C) {
+	fmt.Printf("Running: %s\n", check.TestName())
+
+	org, err := vcd.client.GetAdminOrgByName(vcd.config.VCD.Org)
+	check.Assert(err, IsNil)
+	check.Assert(org, NotNil)
+
+	adminCatalog, err := org.CreateCatalog(TestRenameCatalog, TestRenameCatalog)
+	check.Assert(err, IsNil)
+	check.Assert(adminCatalog, NotNil)
+
+	task, err := adminCatalog.UploadOvf(vcd.config.OVA.OvaPath, TestUploadOvf, TestUploadOvf, 1024)
```

Comment:
'TestUploadOvf` should be named after the test itself rather than using a global variable.

Reply:
Fixed with `check.TestName()+"_ovf"|"_media"`